modules/3d/include/opencv2/3d/detail/submap.hpp

```diff
@@ -195,7 +204,7 @@ class SubmapManager
     Ptr<SubmapT> getCurrentSubmap(void) const;
 
     int estimateConstraint(int fromSubmapId, int toSubmapId, int& inliers, Affine3f& inlierPose);
-    bool updateMap(int _frameId, Ptr<OdometryFrame> _frame);
+    bool updateMap(int _frameId, OdometryFrame _frame);
```

Comment:
> `_`

Avoid using of underscored parameters in public headers.
They are part of documentation and bindings.

Reply:
done